src/app/components/pages/codebase/codebase.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Add some tests to make sure the file being opened is actually being displayed correctly